README.md

```diff
@@ -4,7 +4,9 @@
 This repository is intended for the reference Jenkins configuration as code as well as JobDSL library
 
 ### Getting started
-TBD
+#### Requirements
+* Docker version. Please check your host has docker version 1.11.0. Doecker server and docker client must have the same version 
+
```

Comment:
host = Docker host
add info about compose version
spell check and grammar check
if you do any statements like - this must be like that  then make sure to provide why and what happens if they not